docs/config/tech-docs.yml

```diff
@@ -11,7 +11,7 @@ phase: Beta
 header_links:
   About: /
   Feedback: https://forms.gle/syqb5Fa9RSbo2WC1A
-  Support: /support.html
+  System Admin: /support/providers
```

Comment:
Maybe on the rails end do a `/support/system_admin` endpoint or something as a redirect...

Just food for thought... this whole middleman + rails combo looks error prone